master/internal/resourcemanagers/resource_pool.go

```diff
@@ -344,12 +356,42 @@ func (rp *ResourcePool) Receive(ctx *actor.Context) error {
 func (rp *ResourcePool) receiveAgentMsg(ctx *actor.Context) error {
 	switch msg := ctx.Message().(type) {
 	case sproto.AddAgent:
-		ctx.Log().Infof("adding agent: %s", msg.Agent.Address().Local())
+		agentID := msg.Agent.Address().Local()
+		ctx.Log().Infof("adding agent: %+v", agentID)
 		rp.agents[msg.Agent] = true
+		err := rp.db.AddAgent(&model.AgentStats{
+			ResourcePool: rp.config.PoolName,
+			AgentID:      &agentID,
+			Slots:        msg.Slots,
+			StartTime:    time.Now().UTC().Truncate(time.Millisecond),
```

Comment:
postgres has microsecond precision, maybe use it instead of milliseconds: https://www.postgresql.org/docs/current/datatype-datetime.html

or even better solution: we can use `CURRENT_TIMESTAMP` on postgres side instead of generating it on golang side.

Reply:
done